tests/static/js/testem/coverage.js

```diff
@@ -0,0 +1,18 @@
+/* globals Testem, window, XMLHttpRequest */
```

Comment:
What environmental requirements does this file have?
Copyright/Licence statement?
What is expected to be at the URL /coverage?